src/main/java/se/sundsvall/emailreader/integration/messaging/MessagingClient.java

```diff
@@ -7,12 +7,15 @@
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
 
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+
 import generated.se.sundsvall.messaging.EmailRequest;
+import io.swagger.v3.oas.annotations.Parameter;
 
 @FeignClient(name = CLIENT_ID, url = "${integration.messaging.base-url}", configuration = MessagingConfiguration.class)
 public interface MessagingClient {
 
 	@PostMapping("/{municipalityId}/email")
-	ResponseEntity<Void> sendEmail(@PathVariable final String municipalityId, EmailRequest request);
+	ResponseEntity<Void> sendEmail(@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId, EmailRequest request);
```

Comment:
I know i commented om the client at first, (but that was a mistake from my side).
The client will not produce any API-documents, so the swagger annotation is not useful here.